modules/vector-search/pages/vector-search-index-architecture.adoc

```diff
@@ -0,0 +1,151 @@
+= Vector Search Index Architecture
+:page-topic-type: concept
+:description: Vector Search indexes use features from traditional Search indexes, with unique indexing algorithms and features that allow you to compare vectors in nearest neighbor searches.
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+A Vector Search index still relies on <<sync,>> and uses <<segments,>> to manage merging and persisting data to disk in your cluster.
+All changes from Database Change Protocol (DCP) and the Data Service are introduced to a Search index in batches, which are further managed by segments. 
+
+[#sync]
+== Synchronization with Database Change Protocol (DCP) and the Data Service
+
+The Search Service uses batches to process data that comes in from xref:server:learn:clusters-and-availability/intra-cluster-replication.adoc#database-change-protocol[DCP] and the xref:cloud:clusters:databases.adoc#couchbase-services[Data Service].
+DCP and Data Service changes are introduced gradually, based on available memory on Search Service nodes, until reindexing operations for an index are complete.
+
+The Search Service can merge batches into a single batch before they're sent to the disk write queue, to reduce the resources required for batch processing. 
+
+The Search Service maintains index snapshots on each Search index partition.
+These snapshots contain a representation of document mutations on either a write queue, or in storage.
+
+If the Search Service loses connection to the Data Service, the Search Service compares its rollback sequence numbers in its snapshots with the Data Service when the connection is reestablished.
+If the index snapshots on the Search Service are too far ahead, the Search Service performs a full rollback to get back in sync with the Data Service. 
+
+[#segments]
+== Search Index Segments
+
+Search and Vector Search indexes in Couchbase Server are built with segments. 
+
+All Search indexes contain a root segment, which includes all data for the Search index but excludes any segments that might be stale.
```

Comment:
- There is no such thing as a "root" segment - there is only a collections of segments.
- Data in newer segments can overwrite data in older segments.
- An index snapshot is an abstraction that maps to a subset of segments. We have a **root** index snapshot which maps to all the segments that hold the latest version of data.
- There can exist older index snapshots which are time spaced to support partial rollback capability.

So I think, you meant to say ..
```
All search indexes contain a root index snapshot, which maps to all segments containing the latest version of data.
```